pkg/model/dev.go

```diff
@@ -994,12 +994,13 @@ func (dev *Dev) ToTranslationRule(main *Dev, reset bool) *TranslationRule {
 			)
 		}
 		for _, sync := range dev.Sync.Folders {
+			subpath := getSubPathFromLocalPath(sync.LocalPath)
 			rule.Volumes = append(
 				rule.Volumes,
 				VolumeMount{
 					Name:      main.GetVolumeName(),
 					MountPath: sync.RemotePath,
-					SubPath:   main.getSourceSubPath(sync.LocalPath),
+					SubPath:   main.getSourceSubPath(subpath),
```

Comment:
Is it ok to pass an empty string here? If `getSubPathFromLocalPath` fails, we are not handling the error and instead passing "" here

Reply:
Yes, it will fail when creating the new deployment. But we won't never send an empty string, if it can't detect if it's relative, we will send the latest element of the path